tsdb/engine/tsm1/cache.go

```diff
@@ -180,37 +187,79 @@ func (c *Cache) Keys() []string {
 
 // Values returns a copy of all values, deduped and sorted, for the given key.
 func (c *Cache) Values(key string) Values {
```

Comment:
Totally depends on their settings and how many series they have. For instance if they set it to not take snapshots until 1GB of memory is used and they only feed data into 1 series, it could be that much.